sklearn/hpsearch/gke_bayes_search.ipynb

```diff
@@ -0,0 +1,688 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Everything up to this point is what you would do when training locally.  With larger amount of data it would take much longer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train on Google Container Engine"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up for training on Google Container Engine\n",
+    "\n",
+    "Before we can start training on the Container Engine we need to:\n",
+    "\n",
+    "- Build the Docker image which will be handling the workloads.\n",
+    "- Create a cluster.\n",
+    "\n",
+    "For these we will first set up some configuration variables."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Your Google Cloud Platform project id."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project_id = 'YOUR-PROJECT-ID'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A Google Cloud Storage bucket belonging to your project created through either:\n",
+    "- gsutil mb gs://YOUR-BUCKET-NAME; or\n",
+    "- https://console.cloud.google.com/storage\n",
+    "\n",
+    "This bucket will be used for storing temporary data during Docker image building, for storing training data, and for storing trained models.\n",
+    "\n",
+    "This can be an existing bucket, but we recommend you create a new one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bucket_name = 'YOUR-BUCKET-NAME'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pick a cluster id for the cluster on Google Container Engine we will create.  Preferably not an existing cluster to avoid affecting its workload."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster_id = 'YOUR-CLUSTER-ID'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Choose a name for the image that will be running on the container."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_name = 'YOUR-IMAGE-NAME'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Choose a zone to host the cluster.\n",
+    "\n",
+    "List of zones: https://cloud.google.com/compute/docs/regions-zones/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "zone = 'us-central1-b'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Change this only if you have customized the source."
```

Comment:
Is this referring to the source directory?

Reply:
yes - the user needs to change this only they have changed the content of the `source/` directory, and renamed it it something else.